pkg/controller/generic/transform/options.go

```diff
@@ -96,3 +97,10 @@ func WithOnShutdownCallback(onShutdownCallback OnShutdownCallback) ControllerOpt
 		o.onShutdownCallback = onShutdownCallback
 	}
 }
+
+// WithOutputKind sets main output resource kind.
+func WithOutputKind(kind controller.OutputKind) ControllerOption {
```

Comment:
is the default still `OutputExclusive`?

Reply:
`OutputExclusive` is 0, so when unset it's going to default to it.
But I can set it explicitly, let me change that.